packages/ember-runtime/lib/system/core_object.js

```diff
@@ -754,13 +737,14 @@ let ClassMixinProps = {
     @param [arguments]*
     @public
   */
-  create(...args) {
+  create(props, extra) {
     let C = this;
-    if (args.length > 0) {
-      this._initProperties(args);
-    }
 
-    return new C();
+    if (extra === undefined) {
+      return new C(props);
+    } else {
+      return new C(flattenProps(this, arguments));
```

Comment:
Please update this to be `flattenProps.apply(this, arguments)`.  Passing the `arguments` into another function (other than via `Function.prototype.apply`) is not good...

Reply:
I think it was actually this way before and I changed it back, I need to learn more about the JS perf does and don'ts 😅